### DISCUSSION TITLE
More detailed instructions for setup

### DISCUSSION THREAD

**Discussion by cdistler** [aws/credentials-fetcher#74] [Q&A]
Is there more detailed user instructions for setting up credential-fetcher after it has been installed on AL2023 and I have a gMSA account.

**Comment by saikiranakula-amzn** [comment]
Hi cdistler, Thank you for reaching out. We have a blog post detailing about the use of gMSA on Linux - https://aws.amazon.com/blogs/opensource/aws-now-supports-credentials-fetcher-for-gmsa-on-amazon-linux-2023/. Please let us know if you have any further questions.

**Reply by cdistler** [reply]
I have read the blog post.  I am trying to connect from AL 2023 EKS worker nodes via a smb mount to netapp using credential-fetcher to perform the Kerberos functions.  I have credential-fetcher installed on a worker node, along with a gMSA account, a credspec file but, do not know how to make that work in our scenario.  I just don't see our scenario in that documented and how-to implement it.

**Reply by saikiranakula-amzn** [reply]
We would like to know authentication use-case more clearly to help you with navigate further. Can you please let us know if you are available a call to discuss details? Thank you.

**Reply by cdistler** [reply]
We are creating have the SMB CSI Driver for Kubernetes installed on our EKS cluster and we are using the smb driver to create the PV to netapp.  The problem we are having to recreate the kerberos ticket every 6 days as we are not on a domain.  A call over our issue would be great.

**Reply by saikiranakula-amzn** [reply]
Thank you, cdistler. We will send out an invitation. Can you please share the email and preferred day/time for the call? We are based out of the PST time zone.

**Reply by smhmhmd** [reply]
@cdistler 

You can send to me samiull at amazon dot com and saakla at same domain.

**Reply by smhmhmd** [reply]
https://github.com/aws/credentials-fetcher?tab=readme-ov-file#examples

**Comment by smhmhmd** [comment]
@cdistler 
If you need further help, we can arrange a call.

**Comment by smhmhmd** [comment]
@cdistler 
We have a release for Amazon Linux 2 here, thanks to @awsjohns, @saikiranakula-amzn, @ramapullareddyc, @sb-ruisms, @sreenaath-amzn 
  https://github.com/aws/credentials-fetcher/tree/no-grpc-mode-master/rpm

Doc is at https://github.com/aws/credentials-fetcher/blob/no-grpc-mode-master/README-NOGRPC.MD
Please let us know.
